### PR TITLE
Mailchimp newsletter

### PR DESCRIPTION
**Pull request by ribeirojose**

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -4,9 +4,52 @@ defaultContentLanguageInSubdir = true
...
 title = "Sol Nascente"
-theme = "hugo-now-ui"
+theme = ["hugo-now-ui"]
 staticDir = ["static"]
 
+[menu]
+
+ [[menu.main]]
+    identifier = "market"
+    name = "market landscape"
```
Comment (dokterbob) [inline config.toml]: These have to be translatable & translated.

File: config.toml [hunk 2]
```diff
@@ -17,3 +60,9 @@ staticDir = ["static"]
   [languages.es]
     weight = 3
     languageName = "Español"
+
+[params]
+  instagram = "https://instagram.com/#"
+  facebook = "https://facebook.com/#"
```
Comment (dokterbob) [inline config.toml]:
```suggestion
  facebook = "https://facebook.com/solnascente.eu"
```

File: config.toml [hunk 3]
```diff
@@ -17,3 +60,9 @@ staticDir = ["static"]
   [languages.es]
     weight = 3
     languageName = "Español"
+
+[params]
+  instagram = "https://instagram.com/#"
```
Comment (dokterbob) [inline config.toml]:
```suggestion
  instagram = "https://instagram.com/solnascente.eu/"
```

File: layouts/partials/nav.html [hunk 4]
```diff
@@ -1,52 +1,83 @@
...
+              <span>{{ .Name }}</span>
+            </a>
+          </li>
+          {{ end }}
+        {{ end }}
+      </ul>
+  <!-- Sidebar social network icons -->
+    <ul class="navbar-nav my-auto visible-xs align-items-center">
+      <li>
+        {{ with .Site.Params.mastodon }}<a target="_blank" href="{{ . }}" title="{{ . }}"><i class="fab fa-mastodon fa-2x mx-2"></i></a>{{ end }}
```
Comment (dokterbob) [inline layouts/partials/nav.html]:
```suggestion
        {{ with .Site.Params.mastodon }}<a target="_blank" href="{{ . }}" title="Mastodon"><i class="fab fa-mastodon fa-2x mx-2"></i></a>{{ end }}
```

Give it up for the blind folks!!

Comment (dokterbob) [reply layouts/partials/nav.html]: Also makes the `{{ with }}` unnecessary and thus your code more readable.

File: layouts/partials/nav.html [hunk 5]
```diff
@@ -1,52 +1,83 @@
...
+        <span class="navbar-toggler-bar bar2"></span>
+        <span class="navbar-toggler-bar bar3"></span>
+      </button>
+    </div>
+    <div class="collapse navbar-collapse justify-content-end" id="navigation"
+      data-nav-image="{{ "img/blurred-image-1.jpg" | relURL }}">
+      <ul class="navbar-nav my-4 mr-auto">
+        {{ range .Site.Menus.main }}
+          {{ if .HasChildren }}
+          <li class="nav-item dropdown">
```
Comment (dokterbob) [inline layouts/partials/nav.html]: Maybe I'm reading over it. But I don't see here how the current menu item is active.

**Comment by dokterbob** [comment]
Seems to be some CSS missing or something. ssp_temp_capture

File: config.toml (hunk 1 above)
Comment (ribeirojose) [reply config.toml]: Done.

File: layouts/partials/nav.html (hunk 4 above)
Comment (ribeirojose) [reply layouts/partials/nav.html]: Ok!

File: layouts/partials/nav.html (hunk 5 above)
Comment (ribeirojose) [reply layouts/partials/nav.html]: This is pretty much unnecessary if you don't plan to have dropdown items in your menu.

File: config.toml (hunk 2 above)
Comment (ribeirojose) [reply config.toml]: Done.

**Pull request closed without merging**